### DISCUSSION TITLE
예약 신청 페이지 이동 목록 생성 문의

### DISCUSSION THREAD

**Discussion by ekdclsrn** [changdoc/naver-vaccine-macro#81] [Q&A]
graphql의 결과창에서 response 을 복사해서 병원목록 추출을 했는데

깁밥의원의 response를 붙여넣고 병원 목록 추출을 누르면
1. 김밥의원 120m 열기

이렇게 나타하는데요

다른 병원을 추가하려고 
참치의원의 response를 붙여넣고 병원 목록 추출을 누르면

1번이였던 김밥의원이 사라지고 
1. 참치의원 160m 열기 

이렇게 추가는 안되는데 방법좀 알려주세요.

**Comment by changdoc** [comment]
붙여넣으신 response에 병원 데이터가 하나씩만 들어 있는거 아닌지 확인해보시면 될 것 같습니다.

response에서 복사하시기 전에 preview 탭에서 병원 목록이 원하는 만큼 존재하는지 확인 해보시면 될것 같아요.
![image](https://user-images.githubusercontent.com/1879494/126660004-f801a230-69b4-4da1-93a0-6855b58deb9e.png)

**Reply by ekdclsrn** [reply]
거주 지역과 근무지역을 나눠서 추가하려고 했거든요 ^^

각각 나눠서 저장하는건 불가능한거죠 .

올려주신 방법으로 주소 따서 즐겨찾기 저장하도록 하겠습니다.

늦은시간에 답변 감사합니다.

**Reply by changdoc** [reply]
네 나눠서 리스트를 만들고 싶으시면 각각 지역의 json을 직접 합쳐주는 작업을 하셔야할것 같습니다.
response를 그대로 복사하면 개행 처리가 안되어 내용 파악이 힘드실텐데요.

https://jsonformatter.curiousconcept.com/ 와 같은 서비스를 이용하시면 
데이터 구조를 파악하실수 있을것이고 
필요한 병원 정보만 items 하위에 위치 시키시면 원하는 목록을 만들수 있을 것 같습니다.

**Reply by ekdclsrn** [reply]
그냥 거주지는 이동목록으로 하고 근무지는 주소따서 사용하도록 하겠습니다. 

감사합니다.
